package.json

```diff
@@ -29,15 +29,17 @@
     "babel-cli": "^6.24.1",
     "babel-core": "^6.22.1",
     "babel-eslint": "^7.2.3",
-    "babel-loader": "^6.2.10",
+    "babel-loader": "^7.1.4",
     "babel-plugin-transform-runtime": "^6.22.0",
+    "babel-preset-env": "^1.6.1",
     "babel-preset-es2015": "^6.22.0",
     "babel-preset-stage-0": "^6.22.0",
     "css-loader": "^0.26.1",
     "eslint": "^3.18.0",
     "eslint-loader": "^1.7.0",
-    "extract-text-webpack-plugin": "^2.0.0-rc.3",
+    "extract-text-webpack-plugin": "^3.0.2",
     "file-loader": "^0.10.0",
+    "fork-ts-checker-webpack-plugin": "^0.4.1",
```

Comment:
👏 looks good, is this introduced for better performance? I had a couple of seconds shaved off my build time on this branch, don't know if that's due to the TS process fork.

Reply:
yes, that's for sure. It does the type checking async, not causing the build to fail on type errors